components/upload/UploadList/index.tsx

```diff
@@ -161,7 +161,9 @@ const InternalUploadList: React.ForwardRefRenderFunction<unknown, UploadListProp
 
   const listClassNames = classNames({
     [`${prefixCls}-list`]: true,
-    [`${prefixCls}-list-${listType}`]: true,
```

Comment:
Mybe `listType` lost `text`:

<img width="708" alt="image" src="https://user-images.githubusercontent.com/6134547/215421112-85adac48-5968-4792-a55d-bbfe2e66a70d.png">

Reply:
This is because of this change (I changed it because of your previous suggestion):
```typescript
  const listClassNames = classNames({
    [`${prefixCls}-list`]: true,
    [`${prefixCls}-list-picture`]: listType === 'picture',
    [`${prefixCls}-list-picture-card`]: listType === 'picture-card',
    [`${prefixCls}-list-picture-circle`]: listType === 'picture-circle',
  });
```
I did  see this and checked all the styles but `ant-upload-list-text` wasn't used anywhere so I didn't add it.

Reply:
Should let it back, right? This may cause a break change for older version user style overrides.

Reply:
Which would be better?
```typescript
  const listClassNames = classNames({
    [`${prefixCls}-list`]: true,
    [`${prefixCls}-list-picture`]: listType === 'picture',
    [`${prefixCls}-list-picture-card`]: listType === 'picture-card',
    [`${prefixCls}-list-picture-circle`]: listType === 'picture-circle',
  });
```
or
```typescript
  const listClassNames = classNames({
    [`${prefixCls}-list`]: true,
    [`${prefixCls}-list-${listType}`]: true,
  });
```

Reply:
Sure
```js
  const listClassNames = classNames({
    [`${prefixCls}-list`]: true,
    [`${prefixCls}-list-${listType}`]: true,
  });
```

`listType` has too many items.

Reply:
I agree with your opinion, it helps unintentional side-effects in the future if only needed classNames are listed.